code/modules/mob/living/carbon/human/human.dm

```diff
@@ -841,6 +842,37 @@
 			to_chat(usr, "Failed to apply DNA Infusion to [src].")
 			log_admin("[key_name(usr)] failed to apply a DNA Infusion to [key_name(src)].")
 
+	if(href_list[VV_HK_TURN_INTO_MMI])
+		if(!check_rights(R_DEBUG))
+			return
+
+		var/result = input(usr, "This will delete the mob, are you sure?", "Turn into MMI") in list("Yes", "No")
+		if(result != "Yes")
+			return
+
+		if(!ishuman(src))
+			to_chat(usr, "This can only be done to humanoid mobs.")
+			return
+
+
+		var/obj/item/organ/internal/brain/target_brain = src.get_organ_slot(ORGAN_SLOT_BRAIN)
+
+		if(!target_brain)
+			to_chat(usr, "This mob has no brain to insert into an MMI.")
+			return
+
+		var/obj/item/mmi/new_mmi = new(src.loc)
+
+		target_brain.Remove(src)
+		new_mmi.force_brain_into(target_brain)
+
+		to_chat(usr, "Turned [src] into an MMI.")
+		log_admin("[key_name(usr)] turned [key_name(src)] into an MMI.")
```

Comment:
```suggestion
		log_admin("[key_name(usr)] turned [key_name_and_tag(src)] into an MMI.")
```

better clarity in the logs when people search by the mob_tag

Reply:
Feexed